package.json

```diff
@@ -22,12 +44,12 @@
     "dev-run": "nodemon --watch ./static/build/main.js --exec \"electron --remote-debugging-port=9223 --inspect .\"",
     "dev:main": "yarn run compile:main --watch --progress",
     "dev:renderer": "yarn run ts-node webpack/dev-server.ts",
+    "compile-library": "env NODE_ENV=production yarn run webpack --config webpack/library-bundle.ts",
     "compile": "env NODE_ENV=production concurrently yarn:compile:*",
     "compile:main": "yarn run webpack --config webpack/main.ts",
     "compile:renderer": "yarn run webpack --config webpack/renderer.ts",
     "compile:extension-types": "yarn run webpack --config webpack/extensions.ts",
     "compile:node-fetch": "yarn run webpack --config ./webpack/node-fetch.ts",
-    "postinstall": "yarn run compile:node-fetch",
```

Comment:
Why was this removed?

Reply:
`postinstall` does not really work when lens is consumed via a npm package. I need to figure out where this can be executed automatically.

Reply:
Now it's done via `prepare` hook.